src/code-buddys.html

```diff
@@ -103,7 +103,7 @@ <h2 class="header-h2">on Monday — 18.07.2020</h2>
       </main>
       <footer class="footer-grid">
         <div class="nav-bar">
-          <a href="../index.html" class="idol-icon">
+          <a href="/" class="idol-icon">
```

Comment:
Just a '/' is sufficient here, since `index.html` is always the default file to go to